app-cdr/cdrtools/cdrtools-3.02_alpha09-r3.ebuild

```diff
@@ -42,7 +41,6 @@ FILECAPS=(
 cdrtools_os() {
 	local os="linux"
 	[[ ${CHOST} == *-darwin* ]] && os="mac-os10"
-	[[ ${CHOST} == *-freebsd* ]] && os="freebsd"
```

Comment:
I'd leave this one. It's probably kinda-tricky and since it's CHOST-based, you gain nothing from removing it.